scraper.rb

```diff
@@ -82,18 +82,27 @@ def constituencies
 # ----------
 
 new_format_terms = {
-  '2016' => 'https://en.wikipedia.org/wiki/List_of_members_of_the_parliament_of_Iceland',
-  '2013' => 'https://en.wikipedia.org/wiki/List_of_members_of_the_parliament_of_Iceland,_2013%E2%80%9316',
+  '2016' => 'https://en.wikipedia.org/wiki/Template:MembersAlthing2016',
+  '2013' => 'https://en.wikipedia.org/wiki/Template:MembersAlthing2013',
 }
 
-new_format_terms.each do |term, url|
-  page = MembersPageWithAreaTable.new(response: Scraped::Request.new(url: url).response)
-  members = page.members.each { |m| m[:term] = term }
-  # puts members
-  puts "#{term}: #{members.count}"
-  ScraperWiki.save_sqlite(%i(name term), members)
+old_revisions = {
+  '2013' => 'https://en.wikipedia.org/w/index.php?title=Template:MembersAlthing2013&direction=prev&oldid=714280236',
+}
+
+def scrape_new_format_terms(terms)
+  terms.each do |term, url|
+    page = MembersPageWithAreaTable.new(response: Scraped::Request.new(url: url).response)
+    members = page.members.each { |m| m[:term] = term }
+    # puts members
+    puts "#{term}: #{members.count}"
+    ScraperWiki.save_sqlite(%i(name term), members)
+  end
 end
 
+scrape_new_format_terms(new_format_terms)
+scrape_new_format_terms(old_revisions)
```

Comment:
splitting these out to two lists is nice, but I think I'd combine this into a single function call to more clearly show that it's just doing the same thing for all of them.